lib/pid_ng/PID.cpp

```diff
@@ -0,0 +1,41 @@
+// Project includes
+#include "PID.hpp"
+
+namespace cogip {
+
+namespace pid {
+
+double PID::compute(const double error)
+{
+    double p, i, d;
+
+    // Integral term is the error sum
+    integral_term_ += error;
+
+    // Integral limitation
+    if (integral_term_ > integral_term_limit_) {
+        integral_term_ = integral_term_limit_;
+    }
+    else if (integral_term_ < -integral_term_limit_) {
+        integral_term_ = -integral_term_limit_;
+    }
```

Comment:
Here you could also use `std::min` and `std::max`.

Reply:
done